internal/services/sources_service.go

```diff
@@ -0,0 +1,53 @@
+package services
+
+import (
+	"net/http"
+
+	sources "github.com/RHEnVision/provisioning-backend/internal/clients/sources"
+	"github.com/RHEnVision/provisioning-backend/internal/config"
+	"github.com/RHEnVision/provisioning-backend/internal/payloads"
+	"github.com/go-chi/chi/v5"
+	"github.com/go-chi/render"
+)
+
+func ListSources(w http.ResponseWriter, r *http.Request) {
+	client, err := sources.NewClientWithResponses(*config.SourcesURL)
+	if err != nil {
+		renderError(w, r, payloads.NewClientInitializationError(r.Context(), "sources client", err))
+		return
+	}
+	resp, err := client.ListSourcesWithResponse(r.Context(), &sources.ListSourcesParams{}, AddIdentityHeader)
+	if err != nil {
+		// TODO: use the errors supported by the JSON 400 OR 404
```

Comment:
What does this mean? Obviously we want to use 404 when a resource is not found, which is probably not the case for listing operation.

The code you have is good, you return 3rd party error when unknown error occurs which is HTTP 500 which is correct I think. That is the code for "unknown backend errors".